.gitignore

```diff
@@ -1,2 +1,3 @@
-vendor
-composer.lock
+/vendor/
+/composer.lock
```

Comment:
Sure, sorry for sticking 😅  
I just never considered this likely and (out of pure curiosity) I wanted to get more about when should a third party tool be considered as an application. My php-cs-fixer argument being returned against me, I will reconsider the question :)

About the lock file, let's ignore it for now, if it becomes a real need a day we'll advise